python/Ipam-L2-vpc/README.md

```diff
@@ -0,0 +1,67 @@
+
+# Welcome to your CDK Python project!
+
+This is cdk example project for CDK development with Python.
+
+The project provides a solution that I already suggested for the issues reported from aws-cdk issues about using IPAM pool id in the ec2.vpc L2 construct : 
+https://github.com/aws/aws-cdk/issues/21333
+
+The deployement provides a VPC with 2 subnets ( private and public ) on each availability zone ( you could customize your VPC by modifying the vpc_stack.py code  ) 
+The Ipam_stack have a cidr_range for the top level ipam pool as "11.0.0.0/8" and "11.0.0.0/12" for the region ipam pool you can modify the two variable in app.py
+
```

Comment:
Tell the user what the value of this is. Tell the user what they should be able to see/do with this. how can the customer see the two VPCs in each AZ?.